cvmfs/statistics_database.cc

```diff
@@ -331,45 +375,123 @@ StatisticsDatabase::~StatisticsDatabase() {
 }
 
 
-int StatisticsDatabase::StoreStatistics(const perf::Statistics *statistics,
-                                        const std::string &start_time,
-                                        const std::string &finish_time,
-                                        const std::string &command_name,
-                                        const std::string &repo_name) {
-  std::string insert_statement;
-  if (command_name == "ingest" || command_name == "sync") {
-    insert_statement = PrepareStatementIntoPublish(statistics, start_time,
-                                                               finish_time);
-  } else if (command_name == "gc") {
-    insert_statement = PrepareStatementIntoGc(statistics, start_time,
-                                              finish_time, repo_name);
+StatisticsDatabase *StatisticsDatabase::OpenStandardDB(
+  const std::string repo_name)
+{
+  StatisticsDatabase *db;
+  std::string db_file_path = GetDBPath(repo_name);
+  if (FileExists(db_file_path)) {
+    db = StatisticsDatabase::Open(db_file_path, kOpenReadWrite);
```

Comment:
Perhaps we should check here whether reponame in the database is what we expect it to be.